### PR TITLE
Page Number bind to paginator from parent

### PR DESCRIPTION
**Pull request by KirtiGautam**

### CODE REVIEW

File: src/app/netsuite/settings/employee-mappings/employee-mappings.component.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ export class EmployeeMappingsComponent implements OnInit {
   isLoading = true;
   generalSettings: GeneralSetting;
   rowElement: Mapping;
+  pageNumber = 0;
```
Comment (ashwin1111) [inline src/app/netsuite/settings/employee-mappings/employee-mappings.component.ts:29]: :number

File: src/app/netsuite/settings/generic-mappings/generic-mappings.component.ts [hunk 2]
```diff
@@ -21,6 +21,7 @@ export class GenericMappingsComponent implements OnInit {
   isLoading: boolean;
   mappings: MatTableDataSource<Mapping> = new MatTableDataSource([]);
   setting: MappingSetting;
+  pageNumber = 0;
```
Comment (ashwin1111) [inline src/app/netsuite/settings/generic-mappings/generic-mappings.component.ts:24]: :number

**Review by ashwin1111 — APPROVED** [review]

File: src/app/netsuite/settings/employee-mappings/employee-mappings.component.ts (hunk 1 above)
Comment (KirtiGautam) [reply src/app/netsuite/settings/employee-mappings/employee-mappings.component.ts:29]: This gives Typescript linting error. Refer https://github.com/fylein/fyle-netsuite-app/pull/54/commits/c49a428aef6241cbbb59258964df348725dd6e07

File: src/app/netsuite/settings/generic-mappings/generic-mappings.component.ts (hunk 2 above)
Comment (KirtiGautam) [reply src/app/netsuite/settings/generic-mappings/generic-mappings.component.ts:24]: https://github.com/fylein/fyle-netsuite-app/pull/54#discussion_r603915647

File: src/app/netsuite/settings/employee-mappings/employee-mappings.component.ts (hunk 1 above)
Comment (ashwin1111) [reply src/app/netsuite/settings/employee-mappings/employee-mappings.component.ts:29]: okay, can ignore my comment

**Pull request merged**